rest/config.go

```diff
@@ -1439,6 +1439,21 @@ func (sc *ServerContext) _fetchAndLoadDatabase(nonContextStruct base.NonCancella
 	return true, nil
 }
 
+// stampSGVersion looks at each of the GatewayRegistry and updates the Sync Gateway version to start them on
```

Comment:
I think the comment needs editing, since it's currently only doing one bucket/registry at a time.